packages/butternut/package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "rollup-plugin-butternut",
+  "version": "0.1.0",
```

```suggestion
  "version": "1.0.0",
  "publishConfig": {
    "access": "public"
  },
```

Since we're going to require rollup@^1.20.0, that's a breaking change so we increase the major